pyccl/tests/test_pk2d.py

```diff
@@ -179,9 +187,11 @@ def test_pk2d_function():
         a_arr=aarr, lk_arr=np.log(karr), pk_arr=parr, is_logp=False)
     phere = psp.eval(ktest, atest, cosmo)
     assert_allclose(phere, ptrue, rtol=1E-6)
+    dphere = psp.eval_dlogpk_dlogk(ktest, atest, cosmo)
+    assert_allclose(dphere, -1.*np.ones_like(dphere), 6)
 
 
-def test_pk2d_cls():
+def ftest_pk2d_cls():
```

Comment:
why ftest?

Reply:
Hah, good question. I assume I pressed ctrl+f at some point but missed the ctrl and then this line in the diff.